src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+      class="container"
+    >
+      <input
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+        class="search__bar search__bar--big search__bar--icon icon__big"
```

Comment:
The class names should represent the meaning of the content, not the styles or tag names. Consider simplifying the class names to reflect the purpose of the element rather than its appearance or tag name.